src/api/app/views/shared/_package_branch_form.html.haml

```diff
@@ -1,23 +1,26 @@
+- if show_project_field
+  %p
+    %strong Change Branch Destination
+  %p
+    %label{ for: 'target_project' }
+      %strong Branch project name
+    = text_field_tag 'target_project', target_project, size: 80, maxlength: 200
 %p
-  - if show_source_fields
-    %label{ for: 'linked_project' }
-      %strong Name of original project:
-    %br/
-    = text_field_tag 'linked_project', nil, size: 80, id: 'linked_project', required: true
-    %br/
-    %label{ for: 'linked_package' }
-      %strong Name of package in original project:
-    %br/
-    = text_field_tag 'linked_package', nil, size: 80, id: 'linked_package', required: true
-    %br/
-  - else
-    = hidden_field_tag(:linked_project, @project.name)
-    = hidden_field_tag(:linked_package, @package.name)
   %label{ for: 'target_package' }
-    %strong New package name:
+    %strong Branch package name
   = text_field_tag 'target_package', @package.try(:name), size: 80, maxlength: 200
-  %br/
-  = hidden_field_tag 'target_project', target_project.name if target_project
-  %br/
+%p
   = check_box_tag 'current_revision', false
   %label{ for: 'current_revision' } Stay on current revision
+  - if @revision
+    (##{@revision})
+- if show_project_field
+  - if Configuration.cleanup_after_days
+    %p
+      <input type=checkbox id="disable-autocleanup">
```

Comment:
Why are you using HTML instead of HAML? 🤔

Reply:
I don't want it to have  a `name=` attribute so it does not get send as a parameter and did not find a way in HAML do achieve that. Do you have an idea how to do it in HAML?